client/html/pages/panel/include.html

```diff
@@ -0,0 +1,105 @@
+<template name="includeJobVacancy">
+  <section class="hasPaddingTop">
+    <div class="section-body no-margin">
+      {{#if isInRole 'company' 'default-group'}}
```

Comment:
If you think about it, the only user that should have access to this view is the company, so we probably won't need to have this 'if' in here. We will have to handle the access to the whole route, not just a part of it. 

You don't need to change it now, it's something to one future PR ;)